test/functional/wallet_keypool_topup.py

```diff
@@ -25,70 +17,67 @@ def add_options(self, parser):
 
     def set_test_params(self):
         self.setup_clean_chain = True
-        self.num_nodes = 4
-        self.extra_args = [[], ['-keypool=100'], ['-keypool=100'], ['-keypool=100']]
+        self.num_nodes = 1
+        self.extra_args = [['-keypool=100']]
 
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def run_test(self):
-        wallet_path = self.nodes[1].wallets_path / self.default_wallet_name / self.wallet_data_filename
-        wallet_backup_path = self.nodes[1].datadir_path / "wallet.bak"
-        self.generate(self.nodes[0], COINBASE_MATURITY + 1)
+        node = self.nodes[0]
+        self.generate(node, COINBASE_MATURITY + 10)
```

Comment:
Nit: Why did  this change to `COINBASE_MATURITY + 10`

Reply:
Good point. Probably unnecessary, it's a leftover from prev changes.